src/tiff.rs

```diff
@@ -85,3 +86,33 @@ impl<R: Read + Seek> ImageDecoder for TIFFDecoder<R> {
         }
     }
 }
+
+/// Encoder for tiff images
+pub struct TiffEncoder<W> {
+    w: W,
+}
+
+impl<W: Write + Seek> TiffEncoder<W> {
+    /// Create a new encoder that writes its output to ```w```
```

Comment:
Is there a semantic difference in Commonmark Markdown between triple quoted (```these```) and single quoted (`these`) inline blocks?

Reply:
I would suppose not, however I think single quotes look better so I will change it.

Reply:
I always wondered that since I saw it in several code bases. Might be a historic reason or an older spec that had a difference.